charts/kueue/templates/rbac/role.yaml

```diff
@@ -73,21 +73,11 @@ rules:
       - admissionregistration.k8s.io
     resources:
       - mutatingwebhookconfigurations
-      - validatingwebhookconfigurations
```

Comment:
Looking at the PR  https://github.com/kubernetes-sigs/kueue/pull/3908/files# this should not be removed.